### PR TITLE
ExceptionsAssignment

### PR DESCRIPTION
**Pull request by Evgenygroup**
#28

### CODE REVIEW

File: ExceptionsAssignment/BankAccount.java [hunk 1]
```diff
@@ -0,0 +1,16 @@
+package exceptions;
+
+public class BankAccount {
+    String iban;
```
Comment (sergeylukichev) [inline ExceptionsAssignment/BankAccount.java:4]: private final

File: ExceptionsAssignment/Customer.java [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+    public Customer(String name, int age, BankAccount account) {
+        this.name = name;
+        this.age = age;
+        this.account = account;
+    }
+
+    @Override
+    public String toString() {
+        return name;
```
Comment (sergeylukichev) [inline ExceptionsAssignment/Customer.java:16]: dont want to show age and account?

File: ExceptionsAssignment/MockitoTests.java [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+        account3 = new BankAccount("DE123456789");
+
+        customer1 = new Customer("Customer1", 25, account1);
+        customer2 = new Customer("Customer2", 18, account2);
+        customer3 = new Customer("Customer3", 70, account3);
+
+        listOfCustomers = Arrays.asList(customer1, customer2, customer3);
+
+        v.validateAllCustomers(listOfCustomers);
+        verify(printer, times(1)).print("Customer3 IBAN has wrong length");
```
Comment (sergeylukichev) [inline ExceptionsAssignment/MockitoTests.java:99]: verify(...).print() <-- cool!

**Review by sergeylukichev — APPROVED** [review]
👍 

not bad

File: ExceptionsAssignment/MockitoTests.java (hunk 3 above)
Comment (Evgenygroup) [reply ExceptionsAssignment/MockitoTests.java]: Ok,Thanks

File: ExceptionsAssignment/BankAccount.java (hunk 1 above)
Comment (Evgenygroup) [reply ExceptionsAssignment/BankAccount.java:4]: ok
thanks

**Pull request merged**